x/launch/types/key_chain_name_count.go

```diff
@@ -0,0 +1,21 @@
+package types
+
+import "encoding/binary"
+
+var _ binary.ByteOrder
+
+const (
+	// ChainNameCountKeyPrefix is the prefix to retrieve all ChainNameCount
+	ChainNameCountKeyPrefix = "ChainNameCount/value/"
+)
+
+// ChainNameCountKey returns the store key to retrieve a ChainNameCount from the index fields
+func ChainNameCountKey(chainName string) []byte {
+	var key []byte
+
+	chainNameBytes := []byte(chainName)
+	key = append(key, chainNameBytes...)
+	key = append(key, []byte("/")...)
+
+	return key
+}
```

Comment:
```suggestion
func ChainNameCountKey(chainName string) []byte {
	return []byte(chainName+"/")}
}
```

Reply:
I followed your suggestion in the other PR. to much better for reading too, but it's not a blocker for this PR